paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/CompactActionITCaseBase.java

```diff
@@ -87,4 +98,29 @@ protected void checkFileAndRowSize(
         assertThat(files.size()).isEqualTo(fileNum);
         assertThat(count).isEqualTo(rowCount);
     }
+
+    protected void checkLatestSnapshot(
+            FileStoreTable table, long snapshotId, Snapshot.CommitKind commitKind) {
+        SnapshotManager snapshotManager = table.snapshotManager();
+        Snapshot snapshot = snapshotManager.snapshot(snapshotManager.latestSnapshotId());
```

Comment:
Unboxing of 'snapshotManager.latestSnapshotId()' may produce 'NullPointerException'